file.go

```diff
@@ -22,15 +28,18 @@ type ConfigSecret struct {
 
 // Config : configuration for vaultsync
 type Config struct {
-	SourceAddr    string         `json:"vault_source_addr"`
-	SourceAuth    ConfigAuth     `json:"vault_source_auth"`
-	SourceSecrets []ConfigSecret `json:"vault_source_secrets"`
-	SourceToken   string         `json:"vault_source_token"`
-	TargetAddr    string         `json:"vault_target_addr"`
-	TargetToken   string         `json:"vault_target_token"`
+	SourceAuth         ConfigAuth     `json:"source_auth"`
+	SourceSecrets      []ConfigSecret `json:"source_secrets"`
+	SourcePoliciesPath string         `json:"source_policies_path"`
+	TargetAuth         ConfigAuth     `json:"target_auth"`
 }
 
-// GetConfig : gets configuration
+// FilenameWithoutExt : removes extension from filename
+func FilenameWithoutExt(name string) string {
```

Comment:
One-liner only used once. Does it need to be its own function?

Reply:
Will be used more in the future.